docs/openapi-docs/src/test/resources/coproduct/expected_coproduct_independent.yml

```diff
@@ -84,6 +87,7 @@ components:
           type: integer
           format: int32
     Person1:
+      title: Person
```

Comment:
Is it OK that the `title` is different than the final name?

If I understand correctly, the `1` was added by some name conflict resolution strategy. Unfortunately this happens at a different layer than the one where `title` is assigned. I haven't yet explored how hard it would be to keep these values identical.

Reply:
hm maybe that's actually a feature ;) if the names would serve as unique ids, and titles as user-visible labels. But maybe that's not the right intuition - did you manage to find some reference as to what's the intended meaning of a schema's name & title?

Reply:
I guess they are defined [here](https://json-schema.org/draft/2020-12/json-schema-validation#name-title-and-description) and `title` is essentially just a short, human-optimized info, so we're not doing anything against the spec.

I was just slightly worried about the inconsistency that we have here, i.e. we're using schema name as schema title, albeit with some weird exceptions. I also imagined a situation where Swagger UI user tries to find a schema based on the title displayed in a `oneOf` schema and they can't find it.

All of this may be too minor to worry about, but I wanted to confirm if it's OK to ignore it.

Reply:
Hm if it's human-optimized, then the non-deduplicated name makes sense I think. You can try pasting some of the more complicated schemas into swagger editor to see if there's something weird going on. If no, I think we're good :)